rest_framework/fields.py

```diff
@@ -1162,15 +1161,12 @@ def enforce_timezone(self, value):
                     return value.astimezone(field_timezone)
                 except OverflowError:
                     self.fail('overflow')
-            try:
-                dt = timezone.make_aware(value, field_timezone)
-                # When the resulting datetime is a ZoneInfo instance, it won't necessarily
-                # throw given an invalid datetime, so we need to specifically check.
-                if not valid_datetime(dt):
-                    self.fail('make_aware', timezone=field_timezone)
-                return dt
-            except InvalidTimeError:
+            dt = timezone.make_aware(value, field_timezone)
```

Comment:
I think we should move this back to a try catch, and check that the exception's name using `type`, confirming it's equal to `"InvalidTimeError"`.

Reply:
excellent idea

Reply:
I came out with something I find pretty good. The things I am annoyed with is: since we support both presence and absence of pytz in tests, the CI should test both and I doubt it does (and I yet don't see how without rerunning the test suite in a virtualenv with a different install)